### PR TITLE
Sorted jvms by name in resources list

### PR DESCRIPTION
**Pull request by Sharvaribarve**
The JVMs are not sorted in any order, in the resources topology under the group.
Added a comparator, so that the JVM list, is sorted by the JVM name, and is displayed alphabetically.

Before you contribute, please review these guidelines to help ensure a smooth process for everyone.

Thanks.

* Read [how to properly contribute to open source projects on Github][1].
* Fork the project.
* Use a feature branch.
* Write [good commit messages][2].
* Use the same coding conventions as the rest of the project.
* Commit locally and push to your fork until you are happy with your contribution.
* Make sure to add tests and verify all the tests are passing when merging upstream.
* Add an entry to the [Changelog][3] accordingly.
* Please add your name to the [CONTRIBUTORS.md][7] file. Adding your name to the [CONTRIBUTORS.md][7] file signifies agreement to all rights and reservations provided by the [License][4].
* [Squash related commits together][5].
* Open a [pull request][6].
* The pull request will be reviewed by the community and merged by the project committers.

[1]: http://gun.io/blog/how-to-github-fork-branch-and-pull-request
[2]: http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html
[3]: ./CHANGELOG.md
[4]: ./LICENSE
[5]: http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html
[6]: https://help.github.com/articles/using-pull-requests
[7]: ./CONTRIBUTORS.md

### CODE REVIEW

**Review by PeterTeunissen — APPROVED** [review]

File: jwala-persistence/src/main/java/com/cerner/jwala/persistence/jpa/service/impl/JvmCrudServiceImpl.java [hunk 1]
```diff
@@ -359,7 +360,14 @@ public int removeTemplate(final String jvmName, final String templateName) {
     public List<Jvm> getJvmsByGroupName(String groupName) {
         final Query q = entityManager.createNamedQuery(JpaJvm.QUERY_GET_JVMS_BY_GROUP_NAME);
         q.setParameter(JpaJvm.QUERY_PARAM_GROUP_NAME, groupName);
-        return buildJvms(q.getResultList());
+        List<Jvm> listOfJvms=buildJvms(q.getResultList());
```
Comment (jeddcuison) [inline jwala-persistence/src/main/java/com/cerner/jwala/persistence/jpa/service/impl/JvmCrudServiceImpl.java]: Can you just sort via the name query ? 

e.g. in JpaJvm

    @NamedQuery(name = JpaJvm.QUERY_GET_JVMS_BY_GROUP_NAME, query = "SELECT j FROM JpaJvm j WHERE j.groups.name = :groupName ORDER BY j.name"),

**Review by jeddcuison — CHANGES_REQUESTED** [review]
Since you sorted it in the CRUD level, I think just using ORDER BY in the name query is the less evasive approach.

File: jwala-persistence/src/main/java/com/cerner/jwala/persistence/jpa/service/impl/JvmCrudServiceImpl.java (hunk 1 above)
Comment (Sharvaribarve) [reply jwala-persistence/src/main/java/com/cerner/jwala/persistence/jpa/service/impl/JvmCrudServiceImpl.java]: Ok will do it

Comment (Sharvaribarve) [reply jwala-persistence/src/main/java/com/cerner/jwala/persistence/jpa/service/impl/JvmCrudServiceImpl.java]: Done

**Review by jeddcuison — APPROVED** [review]

**Pull request merged**